capturable/src/commonMain/kotlin/dev/shreyaspatil/capturable/controller/CaptureController.kt

```diff
@@ -79,7 +59,7 @@ class CaptureController {
      * @param config Bitmap config of the desired bitmap. Defaults to [Bitmap.Config.ARGB_8888]
      */
     @ExperimentalComposeApi
-    fun captureAsync(config: Bitmap.Config = Bitmap.Config.ARGB_8888): Deferred<ImageBitmap> {
+    fun captureAsync(config: ColorType = ColorType.ARGB_4444): Deferred<ImageBitmap> {
```

Comment:
This will be a API breaking change but I think we can't get a rid of it 🤔, thoughts?

Reply:
No we can't. 
Or maybe We can have a custom class with that same name. And typealias in android. I think it won't break for old lib users then I guess. Need to dig

Reply:
Okay, we'll plan backward compatibility for this, no worries